File: DSA/Selection_Sort.py
print("################# Selection Sort Program ##################")
L1=[]

n=int(input("Enter the Number of Elements to be inserted in the List : "))
for i in range(n):
    Elements=int(input("The Elements in the List are : "))
    L1.append(Elements)
print("The Elements to be Sorted Are as Follows : ")
print(L1)

## The Code(Part) for Selection Sort...
for i in range(n):
    ST=i
    for j in range(i+1,n):
        if L1[j]<L1[ST]:
            ST=j
    L1[i],L1[ST]=L1[ST],L1[i]
            

print("The List After Sorting is : ")
print(L1)

File: OOP/Selection_Sort.cpp
#include<iostream>
using namespace std;

void Selection_Sort(int arr[], int Num)
{
	int i=0;
	int j=0;
	int ST=0;
	int temp=0;
	for(i=0;i<Num;i++)
	{
		ST=i;
		for(j=i+1;j<Num;j++)
		{
			if(arr[j]<arr[ST])
			{
				ST=j;
			}
		}
		temp=arr[i];
		arr[i]=arr[ST];
		arr[ST]=temp;
	}
	cout<<"The Array after Sorting is as Follows : "<<endl;

	for(int i=0;i<Num;i++)
	{
		cout<<arr[i]<<" ";
	}
}

int main()
{
	
	int n;
	int i;
	cout<<"Enter the Number of Elements u want to add in the Array : ";
	cin>>n;
	
	int arr[n];
	
	cout<<"Enter the Elements in the Array : ";
	for(i=0;i<n;i++)
	{
		cin>>arr[i];
	}
	
	for(int i=0;i<n;i++)
	{
		cout<<arr[i]<<" ";
	}
	cout<<"";
	Selection_Sort(arr,n);
	return 0;
}